### DISCUSSION TITLE
Add a copy to clipboard opton

### DISCUSSION THREAD

**Discussion by SySagar** [chakra-ui/chakra-ui#9358] [Ideas] [answered]
### Description

Currently in the code view , there doesn't exist any copy to clip board. Will be helpfull if we add one.

### Problem Statement/Justification

This will ease the process of copying essential command right into cli/editor.

### Proposed Solution or API

none

**Comment by segunadebayo** [comment, marked as answer]
Docs code blocks already have a copy button.
